cli/tests/tobs-cli-tests/main_test.go

```diff
@@ -71,11 +83,16 @@ func TestMain(m *testing.M) {
 
 	uninstallsObs()
 
-	// wait for the uninstall to succeed
-	// this takes 3 mins because in HA mode
-	// we have three 3 Prometheus instances to gracefully shutdown
-	// and to avoid flakiness.
-	time.Sleep(3 * time.Minute)
+	err = test_utils.WaitForPodsReady(context.Background(), NAMESPACE, 15*time.Minute, 0,
```

Comment:
I think this requires some comment or maybe introduce function wrapper (eg `WaitForPodsRemoved`)